lxd/container.go

```diff
@@ -458,6 +464,12 @@ func containerValidDevices(db *db.Cluster, devices types.Devices, profile bool,
 			if m["connect"] == "" {
 				return fmt.Errorf("Proxy device entry is missing the required \"connect\" property.")
 			}
+
+			if (!strings.HasPrefix(m["listen"], "unix:") ||
+				(strings.HasPrefix(m["listen"], "unix:") && strings.HasPrefix(m["listen"][len("unix:"):], "@"))) &&
```

Comment:
Wouldn't `strings.HasPrefix(m["listen"], "unix:@")` achieve the same here?

Reply:
addressed